sw/device/tests/sim_dv/i2c_device_tx_rx_test.c

```diff
@@ -71,6 +71,13 @@ static volatile const uint8_t kI2cByteCount = 0;
 static volatile bool tx_empty_irq_seen = false;
 static volatile bool cmd_complete_irq_seen = false;
 
+/**
+ * these variables store values based on kI2cIdx
+ */
+static uint32_t i2c_irq_fmt_watermark_id;
+static uint32_t i2c_base_addr;
```

Comment:
You don't need a variable to hold this value, the reference i2c points to the base address after it's initialized.

Reply:
I agree. This variable was removed in the latest version